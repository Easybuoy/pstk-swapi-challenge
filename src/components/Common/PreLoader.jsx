import React from 'react';
import { Pulse } from 'react-preloading-component';
import styled from 'styled-components';

const StyledPreLoader = styled.div`
  text-align: center;
  margin: 5rem 0;
`;

export default function PreLoader() {
  return (
    <StyledPreLoader>
      <Pulse color="#CC0000" size={20} />
    </StyledPreLoader>
  );
}
