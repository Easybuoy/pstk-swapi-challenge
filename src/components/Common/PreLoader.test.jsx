import React from 'react';
import { shallow } from 'enzyme';
// import sinon from 'sinon';

import PreLoader from './PreLoader';

describe('<PreLoader />', () => {

        it('renders the Cart component correctly', () => {
            const wrapper = shallow(<PreLoader />);
            expect(wrapper).toMatchSnapshot();
        });
});