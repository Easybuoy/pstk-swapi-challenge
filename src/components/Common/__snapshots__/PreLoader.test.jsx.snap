// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PreLoader /> renders the Cart component correctly 1`] = `
<styled.div>
  <n
    color="#CC0000"
    size={20}
  />
</styled.div>
`;
